frontend/scripts/react-components/shared/search.component.js

```diff
@@ -0,0 +1,221 @@
+import { h, Component } from 'preact';
+import 'styles/components/shared/search.scss';
+import 'styles/components/shared/autocomplete.scss';
+import Downshift from 'downshift/preact';
+import { findAll } from 'highlight-words-core';
+import cx from 'classnames';
+
+import { PROFILE_PAGES_WHITELIST } from 'constants';
+
+import NodeTitleGroup from 'containers/shared/node-title-group-react.container';
+
+export default class Search extends Component {
+
+  static getNodeId(selectedItem) {
+    // TODO: implement dual selection, for now just displays importer
+    const parts = (selectedItem.id + '').split('_');
+    return parts.length > 1 ? parseInt(parts[0]) : selectedItem.id;
+  }
+
+  static isValidChar(key) {
+    return (/^([a-z]|[A-Z]|ñ|Ñ|á|é|í|ó|ú|Á|É|Í|Ó|Ú){1}$/.test(key));
```

Comment:
my keyboard has ß, ö ä an ü, and I just broke your regex ;)
Portuguese alphabet also has ç ã õ and a few others which will also break your regex - and municipality names have all of those.
Rule of thumb: never use your own regexes to do this sort of thing. I'm sure that there's a npm package out there we can use to do this for us.

Reply:
:trollface: will give it a spin

Reply:
Now we rely on `lodash/deburr` so we support:
https://en.wikipedia.org/wiki/Latin-1_Supplement_(Unicode_block)#Character_table
https://en.wikipedia.org/wiki/Latin_Extended-A

Other alphabets not supported but don't think we need to.